airflow/jobs/scheduler_job.py

```diff
@@ -289,7 +290,12 @@ def _executable_task_instances_to_queued(self, max_tis: int, session: Session =
             .filter(not_(DM.is_paused))
             .filter(TI.state == TaskInstanceState.SCHEDULED)
             .options(selectinload('dag_model'))
-            .order_by(-TI.priority_weight, DR.execution_date)
+            .order_by(
+                -TI.priority_weight,
```

Comment:
```suggestion
                TI.priority_weight.desc(),
```

For consistency? Doesn’t really matter.